control.go

```diff
@@ -285,8 +285,6 @@ func (c *controlConn) setupConn(conn *Conn) error {
 	}
 
 	c.conn.Store(ch)
-	c.session.handleNodeUp(host.ConnectAddress(), host.Port(), false)
```

Comment:
I removed this because there was an issue during init where this called `policy.HostUp` which marked the policy as ready but then session init returned an error because `s.pool.Size() == 0`.